src/cares_wrap.cc

```diff
@@ -150,6 +150,7 @@ using node_ares_task_list =
 
 class ChannelWrap : public AsyncWrap {
  public:
+  enum InternalFields { kCount = AsyncWrap::InternalFields::kCount };
```

Comment:
I don’t think you need these definitions for subclasses if you drop `::InternalFields`

Reply:
Yeah, I was going back and forth on these also. Going off @bnoordhuis' original concern -- that if any of these field counts change it could cause weird breakage -- then it might be better to be more precise here, but then again, the additional enums are certainly unnecessary. I'll drop them :-)